source/fuzz/fuzzer_util.cpp

```diff
@@ -1342,7 +1342,47 @@ opt::Instruction* GetLastInsertBeforeInstruction(opt::IRContext* ir_context,
   return CanInsertOpcodeBeforeInstruction(opcode, &*it) ? &*it : nullptr;
 }
 
-}  // namespace fuzzerutil
+uint32_t MaybeGetLocalVariable(opt::IRContext* ir_context, uint32_t type_id,
+                               uint32_t function_id) {
+  uint32_t return_id = 0;
+  auto function = FindFunction(ir_context, function_id);
+
+  // All of the local variable declarations are located in the first block
+  auto block = function->begin();
+  for (auto& instruction : *block) {
+    if (instruction.opcode() != SpvOpVariable) {
+      continue;
```

Comment:
Add a comment saying that `block` might have `OpLine` and/or `OpNoLine` instructions that can be intermixed with `OpVariable`.